administrator/components/com_associations/models/association.php

```diff
@@ -16,5 +16,27 @@
  */
 class AssociationsModelAssociation extends JModelList
 {
-
+	/**
+	 * Method to get the record form.
+	 *
+	 * @param   array    $data      Data for the form.
+	 * @param   boolean  $loadData  True if the form is to load its own data (default case), false if not.
+	 *
+	 * @return  mixed  A JForm object on success, false on failure
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getForm($data = array(), $loadData = true)
```

Comment:
yes ... add association.xml to /model/forms/association.xml and it will auto load the form
see how other components do it.